src/test/Makefile

```
@@ -479,6 +479,25 @@ transponder_ir_unittest_SRC := \
+dronecan_mag_unittest_DEFINES := \
+		ENABLE_DRONECAN=1 \
+		USE_MAG=
+
+dronecan_mag_unittest_INCLUDE_DIRS := \
+		$(DRONECAN_LIBCANARD_DIR)
+
+# The hydration rule for canard.c lives with the dronecan_gnss test above;
+# order every object of this test after it too.
+$(OBJECT_DIR)/dronecan_mag_unittest/dronecan_mag_canard.c.o \
+$(OBJECT_DIR)/dronecan_mag_unittest/io/dronecan/dronecan_mag.c.o \
+$(OBJECT_DIR)/dronecan_mag_unittest/dronecan_mag_unittest.o: | $(DRONECAN_LIBCANARD_DIR)/canard.c
```

checkmake (0.3.2)

[warning] Target "$(OBJECT_DIR)/dronecan_mag_unittest/dronecan_mag_unittest.o" should be declared PHONY.

(phonydeclared)